### DISCUSSION TITLE
Accuracy Challenge should be compatible with Easy

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#31301] [Ideas]
It's my understanding that Accuracy Challenge isn't compatible with Easy because of the way that Easy interacts with mods that force fails. Sudden Death and Perfect allow 2 violations of their criteria before actually causing you to fail because Easy gives additional lives. That behavior was not fixed before lazer plays were ranked, and it's very unlikely that it'll be changed now.

The effect of that chain of events, is that lazer does not have a 'fail condition' which is distinct from a 'running out of hp' condition. The difficulty arises in that Accuracy Challenge is incompatible with No Fail and Easy, despite it being perfectly functional on paper. An interaction with No Fail is suggested [here](https://github.com/ppy/osu/discussions/28709), so I won't repeat that here.

Primarily, Easy is used because it alters CS, HP, OD and AR. Accuracy Challenge is perfectly compatible with Difficulty Adjust, which also does all of those things, but not with Easy, because of the aforementioned extra lives. Allowing these mods to be compatible would likely require introducing a proper fail system, which is separate from running out of HP, but I think that that is a reasonable solution here.

**Comment by Sleepteiner** [comment]
I agree that the Accuracy Challenge mod and EZ should be compatible. When the daily challenge had a week where every day forced EZ, it was quite annoying to not be able to use the Accuracy Challenge mod to make the process of improving my scores more efficient.
